js/util/browser.js

```diff
@@ -9,6 +9,13 @@
  * it's used for tests.
  */
 
+var window = {
+    addEventListener: function() {},
+    removeEventListener: function() {}
+};
+
+exports.window = window;
+
```

Comment:
Gotcha @mourner, I'll add an export for window as browser/browser.js assumes there is already a global 'window' object defined.
Thanks for the hint.